WordPressAuthenticatorTests/Authenticator/PasteboardTests.swift

```diff
@@ -10,52 +10,62 @@ class PasteboardTests: XCTestCase {
     }
 
     func testNominalAuthCode() throws {
-        if #available(iOS 16.0, *) {
-            throw XCTSkip("UIPasteboard doesn't work in iOS 16.0.") // Check https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/696
-        }
-
         guard #available(iOS 14.0, *) else {
             throw XCTSkip("Unsupported iOS version")
         }
 
+        // FIXME: We'll need to find a way to make the test work the new pasteboard rules
+        //
+        // See:
+        // - https://developer.apple.com/forums/thread/713770
+        // - https://sarunw.com/posts/uipasteboard-privacy-change-ios16/
+        // - https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/696
+        XCTExpectFailure("Paste board access has changed in iOS 16 and this test is now failing")
+
```

Comment:
Confirmed this is needed, by testing on an iOS 15.4 simulator 😉 
<img width="1655" alt="image" src="https://user-images.githubusercontent.com/216089/200306618-c03393e2-ac69-4b1f-b18f-73010f888b01.png">

Reply:
Neat technique.

But do we _really_ need it? Yes, the tests will fail on iOS < 16, but how much of a concern is that for us? What are the chances we'll need to run the tests in iOS < 16?

Thinking about it, it might be beneficial _if_ the `UIPasteBoard` changes are such that we'll need different code between iOS < 16 and >= 16. 🤔 I haven't looked at that yet, this might be a good time to do so.